.github/workflows/ci.yml

```diff
@@ -23,6 +23,8 @@ jobs:
       run: ./build/ci/build.sh -a configure
       env:
         BS: ${{ matrix.bs }}
+        # Avoid using liblzma from the Xcode 16 / MacOSX15.0.sdk, which fails RISCV filter tests.
```

Comment:
This is an unhelpful workaround as it means running the test suite locally on my Mac fails

Reply:
Are you also using Xcode 16? I wonder if 16.2 works with this code?

Reply:
Xcode 16.2 with MacOSX15.2.sdk